keylime.conf

```diff
@@ -13,7 +13,7 @@ receive_revocation_port = 8992
 #=============================================================================
 
 # The binding address and port for the agent server
-cloudagent_ip = 127.0.0.1
+cloudagent_ip = 0.0.0.0
```

Comment:
I think the `keylime.conf` file changes were not intentional. Could you confirm?

Reply:
Right, it was not intentional.